docs/src/operate/c8y/custom-fragments.md

```diff
@@ -37,6 +37,8 @@ sudo tedge config set device.type edge_gateway
 
 Additional fragments can be added to the device by either publishing to a give MQTT topic, or via a file based method. Each section describes what data and when to use it.
 
+> In order to display your custom fragments in Cumulocity's Device Management UI, you have to add them to the "Device Data" Widget of your Devices Info Page. See [Device Details Info](https://cumulocity.com/docs/device-management-application/viewing-device-details/#info) for reference.
```

Comment:
Since we're using Docusaurus, you can put this note as an [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) rather than a reply block.

I think a "tip" admonition feels like the right fit here.

```suggestion
:::tip
In order to display your custom fragments in Cumulocity's Device Management UI, you have to add them to the "Device Data" Widget of your Devices Info Page. See [Device Details Info](https://cumulocity.com/docs/device-management-application/viewing-device-details/#info) for reference.
:::
```